src/visitpy/visit_utils/src/builtin/apropos.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+try:
+    import visit
+except:
+    pass
+
+#
+# Return the list of names of all methods where the specified regex
+# matches either in the method name or its doc string or a stringified
+# version of the object. Use a set to accumulate only unique results
```

Comment:
The sentence "Use a set to accumulate ..." doesn't quite make sense to me.

Reply:
Hmm....not sure I see any issue. Can you suggest re-word?

BTW...I am using `set` here as a the name of a Python data structure type as in `retval = set()` and sets only ever have unique entries.

Reply:
How about:

Use a set to accumulate only unique results, converting it to a list before returning it as a result.

Reply:
> Use a set to accumulate only unique results, converting it to a list before returning it as a result.

Done